src/rating-menu/rating-menu.ts

```diff
@@ -24,7 +24,10 @@ export type RatingMenuState = {
   selector: 'ais-rating-menu',
   template: `
     <div
-      [class]="cx()"
+      [ngClass]="[
+        cx(),
+        state.items.length === 0 ? cx('', 'noRefinement') : ''
```

Comment:
is there a `state.canRefine` available here?

Reply:
No: https://github.com/algolia/instantsearch.js/blob/b6ee2596d353ea692d91929759bc2b2fe34945c7/src/connectors/refinement-list/connectRefinementList.js#L359-L371